snowpack/src/build/import-resolver.ts

```diff
@@ -100,6 +100,16 @@ export function createImportResolver({fileLoc, config}: {fileLoc: string; config
     if (config.packageOptions.external?.includes(spec)) {
       return spec;
     }
+
+    // Note: CSS is allowed to play by different rules. If we can locate a file locally, don’t try and resolve an npm module
+    const isCSSImportingCSS =
+      path.extname(fileLoc) === '.css' &&
+      path.extname(spec) === '.css' &&
+      fs.existsSync(path.join(path.dirname(fileLoc), spec));
```

Comment:
This is the requirements:

1. Original file must be `.css` (can’t be `.js` because that interferes with proxy imports)
2. Imported file must be `.css` (if it‘s not, then it‘s likely an npm package)
3. File must exist locally, relative to the original file.

Only if all those conditions are met do we resolve it locally.

Reply:
Also worth pointing out that we are using `fs.existsSync()` now, but since the first two checks happen first, this should never happen for JS files / other files, so it shouldn’t be a performance hit.

Reply:
this works for CSS-CSS imports, but you're going to run into issues with Sass, Less, etc. Those should be following the same spec for how `@import` is resolved, if they create an `@import` in their build result.